zap/src/main/resources/org/zaproxy/zap/resources/Messages.properties

```diff
@@ -2027,6 +2027,7 @@ Only use this feature to ignore FORMs that you know are safe, for example search
 rules.csrf.ignore.attname = The name of an HTML attribute that can be used to indicate that a form does not need an anti-CSRF Token. If 'rules.csrf.ignore.attvalue' is specified then this must also match the attribute's value. If found any related alerts will be raised at INFO level. 
 rules.csrf.ignore.attvalue = The value of an HTML attribute named by 'rules.csrf.ignore.attname' that can be used to indicate that a form does not need an anti-CSRF Token. If found any related alerts will be raised at INFO level.
 rules.domains.trusted = A comma separated list of URL regex patterns. Any URLs that match the patterns will be considered trusted domains and the issues ignored.
+rules.domxss.browserid = The ID of the browser to be used by DOM XSS scan rule.
```

Comment:
How will users find the ID to use? Can we list them or link to somewhere?

Reply:
That will be documented in the help of the add-on. Ideally the rule configuration would be added by the add-on as well but at the moment core does not allow to remove rule configurations.

Reply:
Sounds good. Add something to tell people to look in the help for the add-on ids then?

Reply:
Tweaked.

Reply:
Docs added in zaproxy/zap-extensions#2055.